workflow/rocoto/workflow_xml.py

```diff
@@ -156,11 +157,22 @@ def _write_crontab(self, crontab_file: str = None, cronint: int = 5) -> None:
             replyto = ''
 
         strings = ['',
-                   f'#################### {pslot} ####################',
-                   f'MAILTO="{replyto}"',
-                   f'{cronintstr} {rocotorunstr}',
-                   '#################################################################',
-                   '']
+                  f'#################### {pslot} ####################',
+                  f'MAILTO="{replyto}"'
+                  ]
+       #AWS need 'SHELL', and 'BASH_ENV' defined, or, the crontab job won't start.
+        pw_csp = os.environ.get('PW_CSP')
+        if ( pw_csp in ['aws', 'azure', 'google'] ):
+             strings = np.append(strings,
```

Comment:
use `strings.append` instead of `np.append`

Reply:
removed "np", and using strings.extend worked.